WooCommerce/Classes/POS/Models/Cart+BarcodeScanError.swift

```diff
@@ -121,9 +121,9 @@ extension PointOfSaleBarcodeScanError {
         )
 
         static let incompleteScan = NSLocalizedString(
-            "pointOfSale.barcodeScan.error.incompleteScan",
-            value: "Partial barcode scan",
-            comment: "Error message shown when scan is incomplete."
+            "pointOfSale.barcodeScan.error.incompleteScan2",
```

Comment:
super nit: I think the convention in the code base before I started following it is `*.versionNumber` but it's really a minor detail that is internal to the code base.
```suggestion
            "pointOfSale.barcodeScan.error.incompleteScan.2",
```